scripts/handlers.js

```diff
@@ -0,0 +1,106 @@
+import * as elements from './elements.js';
+import {
+  getQuestionHeight,
+  getTotalHeight,
+  getTotalOfButtonsAndRemoveActiveClass,
+} from './utils.js';
+
+export function handleMobileBtnClick() {
+  elements.mobileNav.classList.toggle('close');
+}
+
+export function handleTeamMemberEnter(e) {
+  const links = e.currentTarget.querySelector('.team__member--links');
+  links.classList.add('active-element');
+}
+
+export function handleTeamMemberOut(e) {
+  const links = e.currentTarget.querySelector('.team__member--links');
+  links.classList.remove('active-element');
+}
+
+export function handlePortafolioImageEnter(e) {
+  const imageInfo = e.currentTarget.querySelector('.portafolio__image--info');
+  imageInfo.classList.add('active-element');
+}
+
+export function handlePortafolioImageOut(e) {
+  const imageInfo = e.currentTarget.querySelector('.portafolio__image--info');
+  imageInfo.classList.remove('active-element');
+}
+
+export function handleQuestionClick(e) {
+  const clickedQuestion = e.currentTarget;
+  const clickedAnswer = clickedQuestion.querySelector('.question__answer');
+  const clickedAnswerHeight = getTotalHeight(clickedAnswer);
+  if (clickedAnswer.classList.contains('open-answer')) {
+    return;
+  }
+  clickedQuestion.classList.add('open-question');
+  clickedAnswer.classList.add('open-answer');
+  const questionHeight = getQuestionHeight(clickedQuestion, clickedAnswer);
+  clickedQuestion.style['max-height'] = `${questionHeight +
```

Comment:
I do need to access this property because I am calculating the max-height dynamically, so I can't have a class with that property in advance, or at least is easier this way.

Reply:
is it necessary to calculate the max height dynamically? it is way better if you use css styles instead of having to calc exact numbers as css uses graphic calculations and javascript uses the processor (the processor is more limited)

Reply:
It is needed for don't overflow the question content and don't have to have the scroll bar inside the element. I wanted to have that effect of growing the element and I didn't want the scroll, so I used with this solution.